pkg/packagekit/package_pkg.go

```diff
@@ -0,0 +1,97 @@
+package packagekit
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+)
+
+type PkgOptions struct {
+	SigningKey string
```

Comment:
the option fields should be private, they're only exposed via the functional args.